docs/pages/getting-started/running-docker.rst

```diff
@@ -0,0 +1,96 @@
+.. _running-docker:
+
+Getting Started on Docker
+#########################
+
+If you already use Docker, the easiest way to get your Nuts Node up and running for development or production is
+using Docker. To use the latest `master` build use `nutsfoundation/nuts-node:master` (for production environments
+it's advisable to use a specific version).
+
+First determine where you put the files required for the Nuts node, which will be mounted into the Docker container.
+
+Node TLS Certificate
+********************
+
+To connect to an existing Nuts network you need a TLS certificate which authenticate your node. For the development network
+you can use the `nuts-network-development-ca` to directly issue a certificate for your node.
+
+```
+git clone https://github.com/nuts-foundation/nuts-development-network-ca
+cd nuts-development-network-ca && ./issue-cert.sh localhost
+```
+
+The example above issues a certificate for `localhost`, replace it with the correct hostname if you want peers to be able
+to connect to your node. Rename the file to `certificate-and-key.pem`.
+
+Note that the Git repository contains the Certificate Authority certificate (`ca.pem`) which will function as truststore.
+Copy this file as `truststore.pem`.
+
+YAML Configuration File
+***********************
+
+Copy the YAML file below and save it as `nuts.yaml`:
+
+```
+datadir: /opt/nuts
+network:
+  trustStoreFile: /opt/nuts/truststore.pem
+  certFile: /opt/nuts/certificate-and-key.pem
+  certKeyFile: /opt/nuts/certificate-and-key.pem
+```
+
+See :ref:`configuration <nuts-node-config>` for more information on what can be configured.
+
+Mounts
+******
+
+Using this guide the following resources are mounted:
+
+- Readonly PEM file with TLS certificate and private key. They can be separate but in this example they're contained in 1 file.
+- Readonly PEM file with TLS truststore for the particular network you're connecting to.
+- Readonly `nuts.yaml` configuration file.
+- Data directory where data is stored.
+
+Docker Compose
+**************
+
+Copy the following YAML file and save it as docker-compose.yaml:
+```
+version: "3.7"
+services:
+  nuts:
+    image: nutsfoundation/nuts-node:master
+    environment:
+      NUTS_CONFIGFILE: /opt/nuts/nuts.yaml
+    ports:
+      - 5555:5555
+      - 1323:1323
+    volumes:
+      - "./certificate-and-key.pem:/opt/nuts/certificate-and-key.pem:ro"
+      - "./truststore.pem:/opt/nuts/truststore.pem:ro"
+      - "./nuts.yaml:/opt/nuts/nuts.yaml:ro"
+      - "./data:/opt/nuts/data:rw"
+```
+
+Start the service:
+```
+docker-compose up
+```
+
+Without Docker Compose
+**********************
+
+If you want to run without Docker Compose you can use the following command:
+
+```
+docker run --name nuts -p 5555:5555 -p 1323:1323 \
+  --mount type=bind,source="$(pwd)"/certificate-and-key.pem,target=/opt/nuts/certificate-and-key.pem,readonly \
```

Comment:
change `pwd` to a some path since windows users don't use `pwd`?

Reply:
Added note